docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -291,3 +294,36 @@ __Example__:
 ```yaml
   annotation: com.android.buzz.MyAnnotation
 ```
+
+### `numShards`
+
+__Description__: Sets the number of separate shards to create. Read more about shard tests on the [Android developer site](https://developer.android.com/training/testing/junit-runner#sharding-tests).
+
+__Type__: *int*
+
+__Example__:
+```yaml
+  numShards: 4
+```
+
+### `shardIndex`
```

Comment:
Are we keeping this in for now? I thought we had a discussion regarding the suite duplication required for this being somewhat onerous and something we should perform on the backend?
cc: @alexplischke

Reply:
Current implementation is still using `shardIndex`. I think we can remove this field once we implement multiply shardIndex. 🤔

Reply:
@sweeneyskirt-sl Correct! But as @tianfeng92 commented already, this change is yet to be implemented :)